index.html

```diff
@@ -3055,6 +3109,43 @@ <h2 id="docconformance">
 &lt;/figure&gt;
 </pre><!-- source: http://www.geocities.com/SoHo/7373/aquatic.htm#fish -->
       </aside>
+      <section class="proposed addition">
+        <h3 id="docconformance-naming">
+          Requirements for use of ARIA attributes to name <!-- and describe --> elements
+        </h3>
+        <!-- <section>
+          <h4>Elements with implicit ARIA roles which prohibit naming</h4> -->
+          <p>
+            Authors MAY use <a data-cite="wai-aria-1.2/#aria-label">`aria-label`</a> and <a data-cite="wai-aria-1.2/#aria-labelledby">`aria-labelledby`</a> attributes to specify <a data-cite="accname-1.2/#dfn-accessible-name">accessible names</a> for elements which have an implicit or explicit ARIA role which allows naming from authors. [[[wai-aria-1.2]]] defines <a data-cite="wai-aria-1.2/#namefromprohibited">roles which cannot be named</a>.
+          </p>
+          <p>
+            Authors MUST NOT specify `aria-label` or `aria-labelledby` on elements that expose an ARIA role which cannot be named. Elements whose implicit ARIA roles prohibit naming from authors are identified in [[[#docconformance]]].
```

Comment:
again, naming from authors, or by authors (it may well be that the "naming from authors" is used in ARIA 1.2, so if so just ignore...but it does read a bit clunky)